source/_integrations/alarm_control_panel.mqtt.markdown

```diff
@@ -43,7 +43,7 @@ mqtt:
 
 The configuration format of manual configured MQTT items has changed.
 The old format that places configurations under the `alarm_control_panel` platform key
-should no longer be used and is deprecated.
+is no longer available.
```

Comment:
We should remove those from the documentation.

Reply:
I agree that is even better, I'll update the PR.

Reply:
I have removed the content about the removed platform format. I left the `#new_format` anchor since the repair still can generate a link with this anchor.
E.g.: https://www.home-assistant.io/integrations/light.mqtt/#new_format